main/src/ca/uwaterloo/flix/api/lsp/provider/CompleteProvider.scala

```diff
@@ -21,10 +21,39 @@ object CompleteProvider {
-    CompletionItem("println", "println", None, None, CompletionItemKind.Keyword, InsertTextFormat.PlainText, List("(", ")")),
+    CompletionItem("as", "as", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("BigInt", "BigInt", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Bool", "Bool", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("case", "case", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.Snippet, Nil),
+    CompletionItem("chan", "chan", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Char", "Char", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("deref", "deref", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("false", "false", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, List(";")),
+    CompletionItem("Float", "Float", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Float32", "Float32", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Float64", "Float64", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Impure", "Impure", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, List("=")),
+    CompletionItem("Int", "Int", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Int8", "Int8", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Int32", "Int32", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Int64", "Int64", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("let", "let", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("namespace", "namespace", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, List("{", "}")),
+    CompletionItem("new", "new", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Nil", "Nil", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, List(";")),
+    CompletionItem("println", "println", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, List("(", ")")),
+    CompletionItem("pub", "pub", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
```

Comment:
If you look through `Parser.scala` you will note there are more modifiers, e.g. `override`, `sealed` etc.  Ensure that you have added *all* of them! :)